Integrations/integration-Zendesk.yml

```diff
@@ -0,0 +1,462 @@
+        md += '---|---|---|---|---|---|---\n';
+        md += resDetails.ticket.id + '|';
+        md += resDetails.ticket.subject + '|';
+        md += resDetails.ticket.created_at + '|';
+        md += resDetails.ticket.description + '|';
+        md += resDetails.ticket.priority + '|';
+        md += resDetails.ticket.recipient + '|';
+        md += resDetails.ticket.status + '\n';
+
+        resComments = sendRequest('GET', 'tickets/' + args.id + '/comments.json');
+
+        md += '### Ticket comments\n';
+
+        for (var i = 0; i < resComments.comments.length; i++) {
+            md += '##### Comment number: ' + (i+1) + '\n';
+            md += '__Created at__: ' + resComments.comments[i].created_at + '\n';
+            md += '__Comment body__\n';
+            md += resComments.comments[i].body + '.\n\  ';
+        }
+
+
+        return ( {'ContentsFormat': formats.json, 'Type': entryTypes.note, 'Contents': res, "HumanReadable": md, "EntryContext": {}} );
+    };
+
+
+    var fetchIncidents = function() {
+        var now = getLastRun();
+        var sertime;
+        var incidents = [];
+        var labels = [];
+
+        if (!now.time) {
+            sertime = getNowTime(-10);
+        } else {
+            sertime = now.time;
+        }
+
+        var att = {query: 'created>' + sertime};
+        res = sendRequest('GET', 'search.json', '', att);
+
+        if (res.results.length > 0) {
+            for (var i = 0; i < res.results.length; i++) {
+                incidents.push({
+                    name: res.results[i].subject,
+                    details: res.results[i]['description'],
+                    labels: labels,
+                    rawJSON: JSON.stringify(res.results[i]),
+                    severity: parseInt(res.results[i].priority)
+                });
+            }
+
+            setLastRun({'time': res.results[0].created_at});
+        }
+
+        return JSON.stringify(incidents);
+    }
+
+
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            var res = sendRequest('GET', 'users/me.json');
+
+            if (res.user.name !== 'Anonymous user' &&
+                res.user.id) {
+                return 'ok';
+            }
+
+            return JSON.stringify(res);
+
+        case 'zendesk-create-ticket':
+            return createTicket();
+
+        case 'zendesk-list-tickets':
+            return listTickets();
+
+        case 'zendesk-update-ticket':
+            return updateTicket();
+
+        case 'zendesk-ticket-details':
+            return getTicketDetails();
+
+        case 'fetch-incidents':
+            return fetchIncidents();
+        default:
+    }
+  type: javascript
+  commands:
+  - name: zendesk-create-ticket
+    arguments:
+    - name: subject
+      description: The subject of the ticket
+    - name: requester_id
+      description: The numeric ID of the user asking for support through the ticket
```

Comment:
@gshriki For this, I need to create a command for each argument that will show all the available IDs for each argument that requires IDs. Do we want to do this? Shouldn't the user know it by himself?